arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -0,0 +1,1548 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "produces": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/vaults/{resourceName}/backupFabrics/{fabricName}/protectionContainers/{containerName}": {
```

Comment:
Sure I very well understand your concern. However, one cannot mess up the spec, for ease of use in testing internally. We target for customer's ease of use. There are different ways in which you can solve this problem.
One thing that can be done is to add a delegating handler to the request pipeline that changes the resourceNamespace to the desired value, before the request is sent. In node.js you would add a filter to the request pipeline to do the same.

Another thing that could be done is to create a manual wrapper on top of the generated code that can be used for internal testing.